app/services/OverseasTransferChargesService.scala

```diff
@@ -39,7 +39,16 @@ class OverseasTransferChargesService @Inject()(pensionSessionService: PensionSes
       pensionIndex match {
         case Some(index) =>
           model.copy(transferPensionScheme = model.transferPensionScheme.updated(index, pensionScheme.copy(ukTransferCharge = Some(question))))
-        case None => model.copy(transferPensionScheme = model.transferPensionScheme ++ Seq(pensionScheme.copy(ukTransferCharge = Some(question))))
+        case None =>
```

Comment:
I think this would have worked better if you created a separate method for updating the latter part, otherwise we'll start to get really big methods again.

Reply:
Not sure I agree with that, existing method is quite small and separating this out to a different method does not give re-usability as I don't imagine it being used anywhere else .